test/test_distributions.py

```diff
@@ -553,6 +608,7 @@ def test_dist_shape(jax_dist, sp_dist, params, prepend_shape):
         )
 
 
+# TODO: prepend_shape not used?
 @pytest.mark.parametrize("prepend_shape", [(), (2,), (2, 3)], ids=str)
```

Comment:
Good catch! I think we can safely remove this.

Reply:
Should I remove it in this PR?

Reply:
Sure, let's remove it. Thank you!

Reply:
Removed